src/utils/filterPeople.ts

```diff
@@ -0,0 +1,28 @@
+import { Person } from '../types/Person';
+
+export const filterPeople = (
+  people: Person[],
+  centuries: string[],
+  query: string,
+  sex: string,
+): Person[] => {
+  let filteredPeople = [...people];
```

Comment:
Filter doesn't mutate an array, so you don't need to create a copy

Reply:
Fixed